src/ORM/FieldType/DBField.php

```diff
@@ -43,6 +45,10 @@
  */
 abstract class DBField extends ModelData implements DBIndexable
 {
+    public function validate(): ValidationResult
```

Comment:
Signature matches DataObject::validate() for consistency

This method should be moved further down the class